src/methods/sep12/collectSep12Fields.ts

```diff
@@ -43,12 +44,42 @@ export const collectSep12Fields = async ({
     );
   }
 
+  const fieldsToCollect = Object.entries(resultJson.fields ?? {}).reduce(
+    (collectResult: any, field: any) => {
+      const [key, props] = field;
+
+      if (
+        !props.status ||
+        [
+          Sep12CustomerFieldStatus.NOT_PROVIDED,
+          Sep12CustomerFieldStatus.REJECTED,
```

Comment:
Do we want to include rejected fields as well?

Reply:
If the customer's status is `REJECTED`, then that is final (the user will never pass). If the customer's status is `NEEDS_INFO`, then individual fields that are `REJECTED` can be re-collected.

It doesn't look like we're checking the customer status other than when the customer is new, so we'll need to adjust the logic here.